test/Performance/README.md

```diff
@@ -23,19 +23,17 @@ Run `Windows PowerShell` as administrator.
 PHPBench is used to run the benchmarks. Visit http://phpbench.readthedocs.io/en/latest/introduction.html to have an idea how the tool works.
 
 ##### 1. Modify lib/connect.php with the test database credentials
-##### 2. Execute run-perf_tests.py
+##### 2. Modify lib/result_db.php with the result database credentials
+##### 3. The number of iterations for each test can be modified in the test itself (e.g., in test/Performance/benchmark/sqlsrv/SqlsrvSelectVersionBench.php). Each bench class in a test, or the parent class that it extends from, has a @Iteration(n) annotation. If you change the number in this annotation, you will change the number of iterations run for this test. By default, most tests are set to 1000 iterations.
+##### 4. Execute run-perf_tests.py. 
 
 ### Windows
-    py.exe run-perf_tests.py -platform <PLATFORM> -iterations <ITERATIONS> -iterations-large <ITERATIONS_LARGE> -result-server <RESULT_SERVER> -result-db <RESULT_DB> -result-uid <RESULT_UID> -result-pwd <RESULT_PWD
+    py.exe run-perf_tests.py -platform <PLATFORM>

```

Comment:
'> tee' doesn't work for Windows and I couldn't find a way to output to both the command prompt and an output file. Since performance tests on Windows are usually run by remote desktop to a perf machine, I've decided to remove the part that output to a file.